backend/src/middleware/error.js

```diff
@@ -8,11 +9,11 @@ module.exports = (err, _req, res, next) => {
   if (err.message === 'Unauthorized') {
     return res.status(403).send(new Error('You are not authorized to use this resource'));
   }
-
   if (err.message.toLowerCase().includes('not found')) {
     return res.status(404).send(new Error(err.message));
   }
```

Comment:
```suggestion

  if (err.message.toLowerCase().includes('not found')) {
    return res.status(404).send(new Error(err.message));
  }
```

Reply:
my father is capable of beating up your father